stdlib/public/core/Integers.swift

```diff
@@ -23,10 +23,89 @@ extension ExpressibleByIntegerLiteral
   }
 }
 
+//===----------------------------------------------------------------------===//
+//===--- AdditiveArithmetic -----------------------------------------------===//
+//===----------------------------------------------------------------------===//
+
+// FIXME: Add doc comment.
+public protocol AdditiveArithmetic : Equatable {
+  /// The zero value.
+  ///
+  /// - Note: Zero is the identity element for addition; for any value,
+  ///   `x + .zero == x` and `.zero + x == x`.
+  static var zero: Self { get }
+
+  /// Adds two values and produces their sum.
+  ///
+  /// The addition operator (`+`) calculates the sum of its two arguments. For
+  /// example:
+  ///
+  ///     1 + 2                   // 3
+  ///     -10 + 15                // 5
```

Comment:
Examples involving the unary negative operator should probably be removed from the documentation for this protocol.

Reply:
Sure, but it was originally on `Numeric`, and `Numeric` did not have `-` either. I'd like for this PR to focus on the implementation (to unblock @stephentyrone's SIMD PR, for example), and write a separate patch to refine documentation.

Reply:
Right. @xwu, can you open a SR bug for this (since we would want to consider it on `Numeric` anyway) and CC me and @natecook1000?

Reply:
SR-9214